src/content/peerconnection/munge-sdp/index.html

```diff
@@ -63,6 +63,8 @@
         <video autoplay muted></video>
         <h2>Offer SDP</h2>
         <textarea></textarea>
+        <h2>DataChannel send</h2>
+        <textarea id="dataChannelSend" disabled>Data sent </textarea>
```

Comment:
No, it was lefter over when I had the counter on the same line (Data sent: ). Will remove.